fugit2.nvim-scm-1.rockspec

```diff
@@ -0,0 +1,37 @@
+package	= 'fugit2.nvim'
+
+version	= 'scm-1'
+
+rockspec_format = '3.0'
+
+source	= {
+	url	= 'git://github.com/SuperBo/fugit2.nvim.git'
+}
+
+description	= {
+	summary	= 'Git plugin for Neovim (based on libgit2)',
+	homepage	= 'https://github.com/SuperBo/fugit2.nvim',
+	license	= 'MIT',
+}
+
+dependencies = {
+	'lua >= 5.1',
+  'nui.nvim',
+  'nvim-web-devicons',
+  'plenary.nvim',
+}
+
+external_dependencies = {
+	GIT2 = {
+		library = 'git2',
+	}
+}
```

Comment:
Can we add gpgme as an optional dependencies. I used that for gpg signing git commit features.

Reply:
luarocks won't download any `external_dependencies`. It just searches for their presence and reports an error if not found (you can tell it where to look).

You can't declare dependencies as optional.